### PR TITLE
feature - interface login

### PR DESCRIPTION
**Pull request by at-hainguyen3**
feature - interface login

### CODE REVIEW

File: MyApp/View/LoginView/LoginViewController.swift [hunk 1]
```diff
@@ -11,23 +11,31 @@ import UIKit
...
     private var viewModel = LoginViewModel()
 
     // MARK: - initData
     override func configData() {
         super.configData()
+        dummyData()
     }
 
-    // MARK: - initDisplay
+    // MARK: - Config UI
```
Comment (at-daiho) [inline MyApp/View/LoginView/LoginViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Comment by at-hieutran** [comment]
LGTM

**Pull request merged**